compiler/Makefile

```diff
@@ -10,54 +10,32 @@ libdir   ?= $(exec_prefix)/lib
 .PHONY: default
 default: stage$(STAGE)
 
-.PHONY: runtime/libskip_runtime64.a
-runtime/libskip_runtime64.a:
-	$(MAKE) -C runtime $(@F)
-
-.PHONY: runtime/libbacktrace/.libs/libbacktrace.a
-runtime/libbacktrace/.libs/libbacktrace.a:
-	$(MAKE) -C runtime libbacktrace
-
-.PHONY: runtime/libskip_runtime32.bc
-runtime/libskip_runtime32.bc:
-	$(MAKE) -C runtime $(@F)
-
 bootstrap/%.ll: bootstrap/%.ll.gz
 	gunzip -c $< > $@
 
 stage%:
 	mkdir -p $@
-	$(MAKE) $@/bin/skc $@/bin/skfmt $@/bin/skargo $@/lib/libskip_runtime64.a $@/lib/libbacktrace.a $@/lib/libskip_runtime32.bc $@/lib/skip_preamble64.ll $@/lib/skip_preamble32.ll
-
-stage%/lib/libskip_runtime64.a: runtime/libskip_runtime64.a
-	mkdir -p $(@D)
-	cp $< $@
-
-stage%/lib/libbacktrace.a: runtime/libbacktrace/.libs/libbacktrace.a
-	mkdir -p $(@D)
-	cp $< $@
-
-stage%/lib/libskip_runtime32.bc: runtime/libskip_runtime32.bc
-	mkdir -p $(@D)
-	cp $< $@
+	$(MAKE) $@/bin/skc $@/bin/skfmt $@/bin/skargo $@/lib/libstd-x86_64-pc-linux-gnu.sklib $@/lib/libstd-wasm32-unknown-unknown.sklib
 
-stage%/lib/skip_preamble64.ll: preamble/preamble64.ll
+# TODO: Build fresh stdlib for stage >=1?
+stage%/lib/libstd-x86_64-pc-linux-gnu.sklib: bootstrap/libstd-x86_64-pc-linux-gnu.sklib
```

Comment:
Good point. This is a bit of a magic leap for now: the (future) normal way of generating `libstd-*.sklib` will be using `skargo` (once it handles sklibs, cf. #163). For now, it is manually generated by invoking `make`+`skc` directly. This will remain a fallback option in the absence of a usable bootstrap (either on new platforms or in the possible case of botched bootstrap). I will update the PR to make it an explicit `make` target to document it.

Regarding `arm64`, you are correct indeed, will add an `arm64` version.

Reply:
Ok, sounds good. Having a `make` target would be good, and useful in additional to documentation I think. For info, I'm thinking about how I will need to adapt the current workflow I use to test each commit on a branch. Currently I do the following:
```
GIT_SEQUENCE_EDITOR=: git rb -ix "dx 'set -a; CXXFLAGS=\"-O0 -mllvm -inline-threshold=0\"; SKARGO_PROFILE=dev; STAGE=B; PATH=$PWD/compiler/stageB/bin:$PWD/compiler/stageA/bin:$PATH; set +a; make clean && make -C compiler clean install && make test'" main
```
where `dx` is my script to execute a command in a running docker container. Hopefully with your changes I will be able to add a step to build the runtime+stdlib and change `install` to `promote`.